tsconfig.json

```diff
@@ -2,8 +2,8 @@
   "compilerOptions": {
     "lib": ["ES6", "DOM"],
     "target": "ES6",
-    "module": "NodeNext",
-    "moduleResolution": "NodeNext",
+    "module": "esnext",
```

Comment:
We've spend quite some time getting the tsconfig down to work with all packages. Tests are failing after these changes:
```
Test Suites: 2 failed, 1 passed, 3 total
Tests:       2 passed, 2 total
```